bin/www

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+const request = require("request");
+const app = require("../src/backend/app");
+const http = require("http");
+const debug = require("debug")("dtr:backend");
+let version = process.env.VERSION;
+let serverPort = 3005;
+let serverHttps = false;
+let serverHost = "0.0.0.0";
+let serverPublic = "davinci-dtr.logicahealth.org";
+let proxyTarget = "https://davinci-crd.logicahealth.org";
+console.log("Version: " + version);
+if (version == "Dev") {
+  serverPort = 3005;
+  serverHttps = false;
+  serverHost = "0.0.0.0";
+  serverPublic = "0.0.0.0";
+  proxyTarget = "http://localhost:8090";
+} else if (version == "Template") {
+  serverPort = process.env.SERVER_PORT;
+  serverHttps = process.env.SERVER_HTTPS;
+  serverHost = process.env.SERVER_HOST;
+  serverPublic = process.env.SERVER_PUBLIC;
+  proxyTarget = process.env.PROXY_TARGET;
+}
+
+
+/**
+ * Get port from environment and store in Express.
+ */
+
+const port = normalizePort(process.env.PORT || serverPort);
+app.set("port", port);
+
+/**
+ * Create HTTP server.
+ */
+
+const server = http.createServer(app);
+
+/**
+ * Listen on provided port, on all network interfaces.
+ */
+
+server.listen(port);
+server.on("error", onError);
+server.on("listening", onListening);
+
+
+app.use("/fhir", function(req, res) {
+    var url = proxyTarget + "/fhir" + req.url;
+    console.log(url);
+
+    req.pipe(request({url:url,  agentOptions: {
+        rejectUnauthorized: false
+        }})).pipe(res);
+    });
+    app.use("/files", function(req, res) {
+    var url = proxyTarget + "/files" + req.url;
+    console.log(url);
+
+    req.pipe(request({url:url,  agentOptions: {
+        rejectUnauthorized: false
+        }})).pipe(res);
+    });
```

Comment:
Now that the app is all on one port being hosted from the backend, we can rely on the backend server to do our proxying for us.  Even better, we no longer have to proxy requests to the database to a different port, which makes things a bit simpler.